proxy/src/telemetry/sensor/tls.rs

```diff
@@ -0,0 +1,3 @@
+pub struct AcceptHandshake {
+    handle: super::Handle,
+}
```

Comment:
I think this file doesn't belong in this PR?

Reply:
oops, thanks for pointing that out!